From 94bceeb621e36f3188c6246a763def8695526578 Mon Sep 17 00:00:00 2001
From: Valentin Korenblit <valentinkorenblit@gmail.com>
Date: Sat, 20 Oct 2018 10:56:23 +0200
Subject: [PATCH] Set proper value for LIBCLC_INCLUDEDIR

LIBCLC_INCLUDEDIR is the location where mesa3d OpenCL implementation
will look for OpenCL "headers" on the target, when building the OpenCL
kernels.

The value returned by pkg-config for includedir is relevant when
cross-compiling, on the build machine. But in this specific case, we
really need a value that is valid on the target.

Those headers are installed by the libclc package in /usr/share so
that they are not removed by Buildroot target-finalize logic.

Signed-off-by: Valentin Korenblit <valentinkorenblit@gmail.com>
---
 configure.ac | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/configure.ac b/configure.ac
index 864dcae..cc2390b 100644
--- a/configure.ac
+++ b/configure.ac
@@ -2429,7 +2429,7 @@ if test "x$enable_opencl" = xyes; then
                     PKG_CONFIG_PATH environment variable.
                     By default libclc.pc is installed to /usr/local/share/pkgconfig/])
     else
-        LIBCLC_INCLUDEDIR=`$PKG_CONFIG --variable=includedir libclc`
+        LIBCLC_INCLUDEDIR="/usr/share"
         LIBCLC_LIBEXECDIR=`$PKG_CONFIG --variable=libexecdir libclc`
         AC_SUBST([LIBCLC_INCLUDEDIR])
         AC_SUBST([LIBCLC_LIBEXECDIR])
-- 
2.7.4

